kotlinx-coroutines-core/jvm/test/channels/BroadcastChannelMultiReceiveStressTest.kt

```diff
@@ -155,4 +160,10 @@ class BroadcastChannelMultiReceiveStressTest(
             if (stop) break
         }
     }
+
+    @Suppress("UNUSED_PARAMETER")
+    private fun println(debugMessage: String) {
+        // Uncomment for local debugging
+        // kotlin.io.println(debugMessage)
```

Comment:
Uncommenting this leads to a crash. The correct version is `println(debugMessage as Any?)`.